OpenEdXMobile/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -1136,29 +1142,34 @@ private void requestAudioFocus() {
     private Runnable subtitleProcessesor = new Runnable() {
         @Override
         public void run() {
-            try{
+            try {
```

Comment:
FYI, there are a LOT of try/catches in the code that were leftover from the first company that built the app. On top of that, a LOT of them do nothing.